src/App/Utilities/AppHelpers.cs

```diff
@@ -196,6 +197,55 @@ public static async Task<string> SendListOptions(ContentPage page, SendView send
             return selection;
         }
 
+        public static async Task<string> AccountListOptions(ContentPage page, AccountViewCellViewModel accountViewCell)
```

Comment:
IMO, a better way would be something not static, like an instance helper that can be called and can be injected wherever we need it. Because if we start doing Unit Testing this will bite us for sure being static

Reply:
Saving this for a tech debt exercise as it's an established pattern